test/go.sum

```diff
@@ -487,6 +489,8 @@ k8s.io/kube-openapi v0.0.0-20231010175941-2dd684a91f00 h1:aVUu9fTY98ivBPKR9Y5w/A
 k8s.io/kube-openapi v0.0.0-20231010175941-2dd684a91f00/go.mod h1:AsvuZPBlUDVuCdzJ87iajxtXuR9oktsTctW/R9wwouA=
 k8s.io/utils v0.0.0-20231127182322-b307cd553661 h1:FepOBzJ0GXm8t0su67ln2wAZjbQ6RxQGZDnzuLcrUTI=
 k8s.io/utils v0.0.0-20231127182322-b307cd553661/go.mod h1:OLgZIPagt7ERELqWJFomSt595RzquPNLL48iOWgYOg0=
+libvirt.org/go/libvirt v1.10003.0 h1:LEoawzuggD6IL5R/XtnBE8wWJx49i7UZ1HcB7p9glwE=
+libvirt.org/go/libvirt v1.10003.0/go.mod h1:1WiFE8EjZfq+FCVog+rvr1yatKbKZ9FaFMZgEqxEJqQ=
```

Comment:
```suggestion
github.com/libvirt/libvirt-go-module v1.10003.0 h1:LEoawzuggD6IL5R/XtnBE8wWJx49i7UZ1HcB7p9glwE=
github.com/libvirt/libvirt-go-module v1.10003.0/go.mod h1:1WiFE8EjZfq+FCVog+rvr1yatKbKZ9FaFMZgEqxEJqQ=
```

Reply:
I tried and that syntax doesn't work because the module name is `libvirt.org/go/libvirt` (as defined by https://github.com/libvirt/libvirt-go-module/blob/master/go.mod#L1)

```
go: github.com/libvirt/libvirt-go-module@upgrade (v1.10003.0) requires github.com/libvirt/libvirt-go-module@v1.10003.0: parsing go.mod:
        module declares its path as: libvirt.org/go/libvirt
                but was required as: github.com/libvirt/libvirt-go-module
```

Even if the syntax is ok, that wouldn't change the module content. The github module is just a read-only fork of the gitlab one.